src/components/navigation/Header.tsx

```diff
@@ -1,5 +1,6 @@
 import React from "react";
 import { Navbar } from "./Navbar";
+
 import "./Header.scss";
 
 export const Header: React.FunctionComponent = () => {
```

Comment:
I think we can delete the Header.tsx and Header.scss files and call the NavBar component directly from App.tsx

Reply:
Removed